src/Command/InstallWebDirCommand.php

```diff
@@ -108,8 +181,75 @@ private function addFiles($webDir)
      */
     private function removeInstallPhp($webDir)
     {
-        if (file_exists($webDir.'/install.php')) {
-            $this->fs->remove($webDir.'/install.php');
+        if (!file_exists($webDir.'/install.php')) {
+            return;
         }
+
+        $this->fs->remove($webDir.'/install.php');
+        $this->io->text('Deleted the <comment>web/install.php</comment> file.');
+    }
+
+    /**
+     * Stores username and password in .env file in the project directory.
+     *
+     * @param InputInterface $input
+     * @param string         $projectDir
+     */
+    private function storeAppDevAccesskey(InputInterface $input, $projectDir)
+    {
+        $user = $input->getOption('user');
+        $password = $input->getOption('password');
+
+        if (null === $password && null === $user) {
+            return;
+        }
+
+        if (true === $input->getOption('no-dev') && (null !== $user || null !== $password)) {
+            throw new \InvalidArgumentException('Cannot set a password in no-dev mode!');
+        }
+
+        if ((null === $password && null !== $user) || (null === $user && null !== $password)) {
+            throw new \InvalidArgumentException('Must have username and password to set the access key.');
+        }
+
+        $accessKey = hash(
```

Comment:
The standard edition clearly states:
> // This check prevents access to debug front controllers that are deployed by accident to production servers.
> // Feel free to remove this, extend it, or make something more sophisticated.

And as we always deploy the entry point and not only "by accident" but on purpose, we IMO definitely should "make something more sophisticated".